Makefile

```diff
@@ -146,6 +150,12 @@ package: out
 install:
 	mkdir -p ${DESTDIR}${BINDIR}
 	cp ./out/$(PACK_BIN) ${DESTDIR}${BINDIR}/
+	@if [ ! -d "$(IMAGE_JSON_DIR)" ]; then \
+		mkdir -p $(IMAGE_JSON_DIR); \
+	fi
+	@if [ ! -f "$(IMAGE_JSON_FILE)" ]; then \
+		echo '{ "interval": { "pulling_interval": "", "pruning_interval": "7d", "last_prune": "" }, "image": {} }' > $(IMAGE_JSON_FILE); \
+	fi
```

Comment:
Is it possible to initialized the `image.json` file during the configuration? I mean when `pack config pull-policy hourly | daily | weekly | interval=<_d_h_m>` is enabled?

Reply:
yea, I think so.

Reply:
@jjbustamante I think the current implementation already has this support, cause when ever we are calling the `ReadImageJSON` function in [pull_policy.go](https://github.com/buildpacks/pack/pull/2075/files/3bd5c2dc25db28e2e773ed00cb02e7f1b4f1ce01#diff-10ca3b182bbdbfb8cc63caaa60de64594482471c61f73144f6ecf03fcdb751bf) we are checking image.json is present or not if not then we create it with a minimal JSON, and eventually whenever we configure `pack config pull-policy hourly | daily | weekly | interval=<_d_h_m>` the `ReadImageJSON` function will be called via `updateImageJSONDuration` function.

So should I remove those changes from the `MAKEFILE` ?

Reply:
Yeah, I think we can then remove it from the make file, and pack itself can take care of initialized the filesystem with everything it needs